schemas.ts

```diff
@@ -48,7 +47,6 @@ const DeviceToCloud = {
     Button: ButtonD2C,
     Device: DeviceD2C,
     Flip: FlipD2C,
-    GPS: GPSD2C,
```

Comment:
Similarly, should there be an entry, `GNSS: GNSSD2C`?

Reply:
There should be eventually, but the cloud doesn't support this yet (using GNSS instead of GPS for app configuration, I mean). Probably should be a separate ticket.